go.mod

```diff
@@ -379,7 +379,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/pquerna/cachecontrol v0.1.0 // indirect
-	github.com/prometheus/client_golang v1.12.1 // indirect
+	github.com/prometheus/client_golang v1.12.2-0.20220318110013-3bc8f2c651ff // indirect
```

Comment:
Since we're not modifying the root packages of the repo, and the `prometheusexporter` package has its own `go.mod` and `go.sum`, why do we need to update the root `go.mod` and `go.sum`? Is that standard for contrib?

Reply:
not quite sure if we made this change after running into some dependency issue, but I removed it, the tests seems to pass.

Reply:
I haven't followed this in the last couple of days, but this change was to get the modifications we made upstream iirc. pretty sure it gets marked as indirect because it's used in the `exporter/prometheusexporter` package, and the top-level `go.mod` imports that, or something like that.